api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/payments/PaymentCycleService.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.dhsc.htbhf.claimant.service.payments;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.dhsc.htbhf.claimant.entity.Claim;
+import uk.gov.dhsc.htbhf.claimant.entity.PaymentCycle;
+import uk.gov.dhsc.htbhf.claimant.repository.PaymentCycleRepository;
+
+import java.time.LocalDate;
+
+@Service
+public class PaymentCycleService {
+
+    private final PaymentCycleRepository paymentCycleRepository;
+    private final Integer cycleDurationInDays;
+
+    public PaymentCycleService(PaymentCycleRepository paymentCycleRepository,
+                               @Value("${payment-cycle.cycle-duration-in-days}") Integer cycleDurationInDays) {
+        this.paymentCycleRepository = paymentCycleRepository;
+        this.cycleDurationInDays = cycleDurationInDays;
+    }
+
```

Comment:
github diff is misleading here. This isn't a new file, it's been modified and moved to a new package. Same goes for the test.